mlflow/tensorflow/__init__.py

```diff
@@ -1174,17 +1174,24 @@ def _patch_implementation(
 
             batch_size = None
             try:
+                single_input_model = isinstance(inst.input_shape, tuple)
```

Comment:
```suggestion
                is_single_input_model = isinstance(inst.input_shape, tuple)
```

to clarify this is a boolean.

Reply:
I'll do the renaming in a single commit 👍